server/src/server.ts

```diff
@@ -410,10 +422,10 @@ async function resolveSettings (
         )
       }
       return await promise.then(
-        path => {
+        async path => {
           let library = path2Library.get(path)
           if (library == null) {
-            library = require(path)
+            library = (await Function(`return import('file://${path.replace(/\\/g, '\\\\')}')`)()).default
```

Comment:
Sadly, TypeScript transpile `import()` into `require()`, even with `"module": "node12"`
When this behavior is fixed, this should be used
``library = (await import(`file://${path}`)).default``
- https://github.com/microsoft/TypeScript/issues/43329#issuecomment-1027238813
- https://github.com/microsoft/TypeScript/issues/46452

Reply:
Indeed, unfortunately, we must do this ugly trick while `typescript` doesn't support that.

We might create a new issue to track that, so we can fix it later when we can.

Reply:
Issue available: https://github.com/standard/vscode-standard/issues/378